tests/v3/covid-19/api_vaccine.spec.js

```diff
@@ -81,6 +81,7 @@ describe('Testing /v3/covid-19/vaccine/ vaccine coverage', () => {
 				res.body.length.should.not.equal(0);
 				res.body[0].should.have.property('country');
 				res.body[0].should.have.property('timeline');
+				res.body[0].should.have.property('timelineList');
```

Comment:
When you make this a query parameter you will need to add new tests using the query parameter and checking the format is correct